app/views/cookie_preferences/show.html.erb

```diff
@@ -146,8 +139,7 @@
 
           <ul>
             <li>
-              remember your progress through a form (for example a licence
-              application)
+              remember your progress through a form (for example, if you sign up to receive our personalised emails)
             </li>
             <li>
               remember the notifications you've seen so we do not show them
```

Comment:
Could you possibly add a full stop after the `Find out more about cookies on this service` link at the end of the page? I appreciate this is missing in production already!

Reply:
And add another full-stop after the privacy notice link!